### ISSUE TITLE
ability to specify default root append the group command incase root not mentioned

### ISSUE THREAD

**Issue by Nilavarasi-Ashika** [perpetualintelligence/cli#54]
**Is your feature request related to a problem? Please describe.**
Need an ability to specify default root value to append with group command in the scenario's like command execution root is not specified.

**Describe the solution you'd like**
Command format
e.g., xxx yyy exCommand
xxx - (Root)
yyy - (group)
In this case user mentioned command like yyy exCommand need and ability to append the root (xxx) which is configured in default.

**Comment by Nilavarasi-Ashika** [comment]
@ChandramouliShridharan @ameyasubhedarats  FYI

**Comment by asperpint** [comment]
@Nilavarasi-Ashika, our terminal framework supports multiple roots. If we provide this option to specify the default root, it may cause issues for other roots.

Are you suggesting this options for Terminals that have one 1 root command.

e.g. -> 
root1 group1 cmd1
root2 group1 cmd2
root3 group3 cmd3

**Comment by Nilavarasi-Ashika** [comment]
@asperpint  yes we are suggesting default value option for terminals that have one root command.

**Comment by prawasu** [comment]
@Nilavarasi-Ashika, thanks for reporting this issue. We are currently investigating a similar functionality requested by another customer. As part of this request, we will link the terminal to the root command. After this, you can execute your terminal commands from the regular command prompt by specifying the root and subcommand.

E.g. dotnet build "test.csproj" executes the dotnet app and send the command to the dotnet app. After linking, your terminal becomes an app (similar to dotnet) that accepts the commands.

**Comment by prawasu** [comment]
Similar to #62
